src/library/librarywatcher.h

```diff
@@ -77,6 +78,14 @@ class LibraryWatcher : public QObject {
   void SetRescanPaused(bool pause);
 
  private:
+  class WatchedDir : public Directory {
+   public:
+    WatchedDir() { active_ = true; }
```

Comment:
Can you use an initialiser list instead of assignment in both these constructors?

Reply:
done